--- debian/foreman/usr/share/foreman/Gemfile  2012-07-14 13:44:26.000000000 +0100
+++ debian/foreman/usr/share/foreman/Gemfile.orig 2012-07-11 19:23:44.000000000 +0100
@@ -5,7 +5,7 @@
 gem 'json'
 gem 'rest-client', :require => 'rest_client'
 gem "audited-activerecord", "3.0.0.rc1"
-gem "has_many_polymorphs", :git => "https://github.com/jystewart/has_many_polymorphs.git", :ref => '03429a61e511f394e9f96af0c8998268ca99d42b'
+gem "has_many_polymorphs", "3.0.0.beta1"
 gem "will_paginate", "~> 3.0.2"
 gem "ancestry", "~> 1.2.4"
 gem 'scoped_search', '>= 2.3.7'
